tutorial/README.md

```diff
@@ -0,0 +1,168 @@
+And `context` is an optional dictionary of parameters (batch_size, learning_rate, etc.) specified at run-time.
+
+After adapting the original model description, we end up with the [following model](./hello/hello_world.py).
+
+```python
+""" hello_world.py
+"""
+import tensorflow as tf
+
+def main(server, log_dir, context):
+    """
+    server: a tf.train.Server object (which knows about every other member of the cluster)
+    log_dir: a string providing the recommended location for training logs, summaries, and checkpoints
+    context: an optional dictionary of parameters (batch_size, learning_rate, etc.) specified at run-time
+    """
+
+    c = tf.constant("Hello, distributed TensorFlow!")
+    sess = tf.Session(server.target)  # Create a session on the server.
+    res = sess.run(c)
+    print(res)
+```
+
+We easily test the final model locally, by using the [run_local.py](./run_local.py) script. Please make sure to [install TensorFlow](https://www.tensorflow.org/install/) beforehand.
+
+```bash
+$ python run_local.py
+2017-11-26 17:00:06.076234: I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:215] Initialize GrpcChannelCache for job local -> {0 -> localhost:51224}
+2017-11-26 17:00:06.076526: I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:324] Started server with target: grpc://localhost:51224
+2017-11-26 17:00:06.080764: I tensorflow/core/distributed_runtime/master_session.cc:1004] Start master session e3b62dd340db772c with config:
+b'Hello, distributed TensorFlow!'
+[2017-11-26 17:00:06,108|__main__|INFO]: Done
+```
+
+In order to make this file available to our cluster, we create an archive of the hello folder and upload it to a location from where it available to the cluster. For example, in our example to [https://downloads.mesosphere.com/tensorflow-dcos/examples/hello.zip](https://downloads.mesosphere.com/tensorflow-dcos/examples/hello.zip).
+
+## Prepare the Options File
+
+With the current deployment model for the beta DC/OS TensorFlow package, you have to specify the model you want to run as options during the package installation (as mentioned earlier, this behavior will change in future releases).
+The easiest way of specifying these options is by installing the package using the [DC/OS CLI](https://docs.mesosphere.com/latest/cli/install/).
+
+```bash
+$ dcos package install beta-tensorflow --options=OPTIONS.json
+```
+
+We provide a [script](../scripts/new-config.sh) which helps you generate a template for such option files.
+
+```bash
+$ scripts/new-config.sh helloworld
+```
+
+This command will generate the following template here `examples/local/helloworld..json`
+
+```json
+{
+  "service": {
+    "name": "helloworld",
+    "job_url": "{{JOB_URL}}",
+    "job_path": "",
+    "job_name": "{{JOB_NAME}}",
+    "job_context": "",
+    "shared_filesystem": "",
+    "use_gcs_key_secret": false,
+    "use_tensorboard": false,
+    "user": "root",
+    "tf_image": "mesosphere/dcos-tensorflow:v1.3",
```

Comment:
These images will be out of date as soon as the new release lands. Should we just remove them entirely? That way new options files will use the default image. This is the approach we are taking with the examples in #10.

If we change this here, we'll also need to actually edit `bin/new-config.sh`.